connexion/utils.py

```diff
@@ -28,14 +31,23 @@
 }
 
 
-def flaskify_endpoint(identifier):
+def flaskify_endpoint(identifier, randomize=None):
     """
     Converts the provided identifier in a valid flask endpoint name
 
     :type identifier: str
+    :param randomize: If specified, add this many random characters (upper case
+        and digits) to the endpoint name, separated by a pipe character.
+    :type randomize: int | None
     :rtype: str
     """
-    return identifier.replace('.', '_')
+    result = identifier.replace('.', '_')
+    if randomize is None:
+        return result
+
+    def generator(size=randomize, chars=string.ascii_uppercase + string.digits):
```

Comment:
No good one, mostly this was too much for me for a single line:

```python
''.join(random.SystemRandom().choice(string.ascii_uppercase + string.digits) for _ in range(randomize))
```

But there's a few other ways it could be broken up, sure.